interop-testing/src/test/java/io/grpc/testing/integration/ProxyTest.java

```diff
@@ -79,13 +82,16 @@ public void smallLatency() throws Exception {
     clientIn.readFully(new byte[5]);
 
     // test
-    long start = System.nanoTime();
-    clientOut.write(message, 0, 1);
-    clientIn.read(message);
-    long stop = System.nanoTime();
-
-    long rtt = (stop - start);
-    assertEquals(latency, rtt, latency);
+    List<Long> rtts = new ArrayList<>();
+    for (int i = 0; i < 3; i++) {
+      long start = System.nanoTime();
+      clientOut.write(message, 0, 1);
+      clientIn.read(message);
+      rtts.add(System.nanoTime() - start);
+    }
+    Collections.sort(rtts);
```

Comment:
What about 
```
long minRtt = Long.MAX_VALUE;
for {
  long rtt = System.nanoTime() - start;
  minRtt = rtt < minRtt ? rtt : minRtt;
}
assertEquals(latency, minRtt, .5 * latency);

Reply:
I could. At times I've put in a `System.out.println(rtts)` and it is nice to see all the values. I'd say this seems like a toss up, where it doesn't matter much which approach is taken.